bin/waagent2.0

```diff
@@ -3246,6 +3246,10 @@ class Certificates(object):
             if not dom.getElementsByTagName(a):
                 Error("Certificates.Parse: Missing " + a)
                 return None
+        certificateFormat = GetNodeTextData(dom.getElementsByTagName("Format")[0])
```

Comment:
Could you please revert the changes in bin/waagent2.0. This is a deprecated version of the Agent.

For your change, you need to make changes in azurelinuxagent/common/protocol/wire.py:1448 - `azurelinuxagent.common.protocol.wire.Certificates#parse` method

I've described below where the Certificates class is used.